meta-lmp-base/recipes-sota/aktualizr/aktualizr_%.bbappend

```diff
@@ -22,13 +23,23 @@ PACKAGECONFIG[dockerapp] = "-DBUILD_DOCKERAPP=ON,-DBUILD_DOCKERAPP=OFF,,docker-a
 PACKAGECONFIG_append_class-target = " dockerapp"
 PACKAGECONFIG_remove_class-target_riscv64 = "dockerapp"
 
-SYSTEMD_PACKAGES += "${PN}-lite"
+PACKAGES += "${PN}-lite-path"
+
+SYSTEMD_PACKAGES += "${PN}-lite ${PN}-lite-path"
 SYSTEMD_SERVICE_${PN}-lite = "aktualizr-lite.service"
+SYSTEMD_AUTO_ENABLE_${PN}-lite ?= "enable"
+SYSTEMD_SERVICE_${PN}-lite-path = "aktualizr-lite.path"
+SYSTEMD_AUTO_ENABLE_${PN}-lite-path ?= "enable"
 
 do_install_append() {
     install -d ${D}${systemd_system_unitdir}
     install -m 0644 ${WORKDIR}/aktualizr-lite.service ${D}${systemd_system_unitdir}/
+    install -d ${D}${systemd_system_unitdir}
```

Comment:
Not needed as we already have a similar line above this one (same path).

Reply:
Nice catch.  Fixed.